src/game_config.cpp

```diff
@@ -434,6 +435,7 @@ void load_config(const config &v)
 		load_attribute(s, "server_message",   server_message);
 		load_attribute(s, "player_joins",     player_joins);
 		load_attribute(s, "player_leaves",    player_leaves);
+		load_attribute(s, "game_created",     game_created);
```

Comment:
Shouldn't you update data/game_config.cfg as well?

https://github.com/wesnoth/wesnoth/blob/808bd59dae43b57ff8f1fa39158af9722191872d/data/game_config.cfg#L97-L108